### PR TITLE
Completed es6-template-literals

### PR DESCRIPTION
**Pull request by Cole-Allen**
image

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = number + street;
```
Comment (scbowler) [inline es6-template-literals/index.html:13]: This should use a template literal

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = fruit + item;
```
Comment (scbowler) [inline es6-template-literals/index.html:28]: This should use a template literal

**Review by scbowler — CHANGES_REQUESTED** [review]
Good start, but a few updates needed. Let me know if you have any questions.

**Comment by Cole-Allen** [comment]
image

File: es6-template-literals/index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number + street}`;
```
Comment (scbowler) [inline es6-template-literals/index.html:13]: OR:
```JAVASCRIPT
`${number}${street}`
```

File: es6-template-literals/index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log(`Bio: ${bio}`);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit + item}`;
```
Comment (scbowler) [inline es6-template-literals/index.html:28]: OR:
```JAVASCRIPT
`${fruit}${item}`
```

**Review by scbowler — APPROVED** [review]
Looks good! 👍 

2 FYI comments, no changes required

**Pull request merged**